projects/arlas-toolkit/assets/i18n/en.json

```diff
@@ -265,5 +265,6 @@
 	"successful": "successful",
 	"failed": "failed",
 	"dissmissed": "dissmissed",
-	"yes": "yes"
+	"yes": "yes",
+	"processing error occured": "An error occured in {{processingName}} service"
```

Comment:
If we start using variable insertion in translate keys, I think we should discuss whether it should be displayed in the key that there is a variable to replace.